twake/backend/node/src/services/channels/web/types.ts

```diff
@@ -51,4 +51,10 @@ export interface ChannelWebsocket {
 export class CreateChannelBody {
```

Comment:
All string types ?
Maybe you could something like ` [key: string]: string;`.

WDYT @chamerling  ?